app/pages/notifications.spec.js

```diff
@@ -30,18 +32,20 @@ describe('Notifications', function() {
     });
 
     it('will notify when no notifications present', function() {
+      const stub = sinon.stub(Notifications.prototype, 'componentWillMount');
       wrapper = mount(<Notifications user={{ id: 1 }} />);
       assert(wrapper.contains(<span>You have no notifications.</span>));
+      stub.restore();
     });
   });
 
   describe('it correctly display projects', function() {
     beforeEach(function () {
       wrapper = shallow(
-        <Notifications user={{ id: 1 }} />,
```

Comment:
So, I took out the user here so `getProjectNotifications` would not run and log the API catch error to the console when the component is mounted, but I could also provide a stub instead as I did above. Running `groupNotifications(testNotifications)` is mocking an example of what might be returned by an API call in `getProjectNotifications`.